pkg/cableengine/syncer/syncer.go

```diff
@@ -23,8 +23,9 @@ type GatewaySyncer struct {
 	version string
 }
 
-const GatewayUpdateIntervalSeconds = 5
-const GatewayStaleTimeoutSeconds = GatewayUpdateIntervalSeconds * 3
+var GatewayUpdateInterval = 5 * time.Second
+var GatewayStaleTimeout = time.Duration(GatewayUpdateInterval.Seconds()) * 3 * time.Second
```

Comment:
Isn't it simpler to keep L27 as *3 ? (it would be time unit independent)

Reply:
yeah - you're right -I made that more difficult - not sure what I was thinking 😄  - I'll change it